scripts/travisci/check_flake8.sh

```diff
@@ -0,0 +1,19 @@
+exit_status=0
+# Absolute verification
+# W191 - Indentation contains tabs
+# E101 - Indentation contains mixed spaces and tabs
+find -iname "*.py" | flake8 --select=E101,W191
+if [ $? -ne 0 ]; then
+  exit_status=1
+fi
+
+# Incremental verification
+# E501 - Line too long
+if [[ "${TRAVIS_PULL_REQUEST}" != "false" ]]; then
+  git diff ${TRAVIS_COMMIT_RANGE} | flake8 --diff --select=E501
+  if [ $? -ne 0 ]; then
+    exit_status=1
+  fi
+fi
+
+exit $exit_status
```

Comment:
default bash behavior is non-zero exit status if any subcommand had non-zero exit status. does flake8 always exit(0)?

Reply:
exit "${exit_status}"

see https://google.github.io/styleguide/shell.xml

Reply:
Done

Reply:
I noticed that in the push build the script for flake8 was returning zero even though the absolute checks did not passed for the test files I added in this PR, and the incremental errors were not verified since TRAVIS_PULL_REQUEST is false for that build.
I will check this again to see what's going on with the return value.